script.js

```diff
@@ -10,3 +10,168 @@ union.addEventListener('click', () => {
+
+  const card1 = document.createElement('div');
+  card1.classList.add('card-1');
+
+  const heading2 = document.createElement('h2');
+  heading2.classList.add('heading-2');
+  heading2.innerHTML = projects[i].name;
+
+  const specify = document.createElement('div');
+  specify.classList.add('specify');
+
+  const canopy = document.createElement('p');
+  canopy.classList.add('card-text-1');
+  canopy.innerHTML = 'Canopy';
+
+  const dot = document.createElement('img');
+  dot.src = './images/dot.png';
+  dot.alt = 'dot';
+
+  const backEndDev = document.createElement('p');
+  backEndDev.classList.add('card-text');
+  backEndDev.innerHTML = 'Back End Dev';
+
+  const dot2 = document.createElement('img');
+  dot2.src = './images/dot.png';
+  dot2.alt = 'dot';
+
+  const t2015 = document.createElement('p');
+  t2015.classList.add('card-text');
+  t2015.innerHTML = '2015';
+
+  specify.appendChild(canopy);
+  specify.appendChild(dot);
+  specify.appendChild(backEndDev);
+  specify.appendChild(dot2);
+  specify.appendChild(t2015);
+
+  card1.appendChild(specify);
+  card1.appendChild(heading2);
+
+  workSection.appendChild(cards);
+  workSection.appendChild(card1);
+
+  const p = document.createElement('p');
+  p.innerHTML = 'A daily selection of privately personalized reads; no accounts or sign-ups required.';
+  card1.appendChild(p);
+
+  const languages = document.createElement('ul');
+
+  for (let j = 0; j < projects[i].technologies.length; j += 1) {
+    const newLi = document.createElement('li');
+    newLi.classList.add('lang');
+    newLi.innerHTML = projects[i].technologies[j];
+    languages.appendChild(newLi);
+  }
+
+  card1.appendChild(languages);
+
+  const seeBtn = document.createElement('button');
+  seeBtn.classList.add('btn', 'see-btn');
+  seeBtn.innerHTML = 'See Project';
+
+  seeBtn.addEventListener('click', () => {
+    document.querySelector('.modal-box').style.display = 'flex';
+    document.body.style.overflow = 'hidden';
+    document.querySelector('.modal-box').innerHTML = `
+    <div class="modal-content">
+    <div class="modal-project-title-and-close">
+      <h3 class="modal-project-title">${projects[i].name}</h3>
+      <img src="images/close-modal.png" alt="close modal button" class="close-modal-btn" onclick="closeModal()">
+    </div>
+    <ul class="modal-project-details">
+      <li>CANOPY</li>
+      <li><img src="images/dot.png" alt="dot icon"></li>
+      <li>Back End Dev</li>
+      <li><img src="images/dot.png" alt="dot icon"></li>
+      <li>2015</li>
+    </ul>
+    <img src=${projects[i].featuredImage} alt="project snapshot" class="modal-project-snapshot">
+    
+    <div class="p-and-btns-box">
+      <p class="modal-primary-text">${projects[i].description}</p>
+      <div class="modal-skills-and-btns">
+        <ul class="modal-project-tags">
+          <li class="modal-project-tag">html</li>
+          <li class="modal-project-tag">css</li>
+          <li class="modal-project-tag">javascript</li>
```

Comment:
- [ ] This is a brilliant attempt :+1: but I think you should focus on implementing the spacing between each element as featured in the Figma design. This will really make your work more aesthetic and visually impressive without any cluttering or overcrowding of information :wink: 

- Here's a [screencast](https://www.awesomescreenshot.com/video/15414558?key=b8a20120b7b7c218a61601b386dace93) from your design as against the Figma design :point_down: 
![image](https://user-images.githubusercontent.com/99715640/223689476-4c4050b3-1170-4efa-bfce-5d178497ab58.png)